### PR TITLE
Fix a bug that incorrectly stores the exchange_token amount

### PR DESCRIPTION
**Pull request by yisiliu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "bignumber.js": "2.0.0",
         "chai-as-promised": "^7.1.1",
         "openzeppelin-solidity": "^3.2.0",
-        "sinon": "^9.2.2",
```
Comment (zhouhanseng) [inline package.json:10]: Removed, since the new coverage test does not work with sinon and sinon is of little value to our test actually.

**Comment by yisiliu** [comment]
13000+ why?

File: README.md [hunk 2]
```diff
@@ -24,7 +24,6 @@ truffle build
 
 To test the project:
 ```
-npm install chai ganache-cli
```
Comment (zhouhanseng) [inline README.md:27]: Don't need to launch ganache-cli separately since we used `truffle run coverage`

Comment (yisiliu) [reply README.md:27]: no what if you want to debug some specific transaction?

Comment (zhouhanseng) [reply README.md:27]: got it

**Pull request merged**